packages/core/src/components/graphs/scatter.ts

```diff
@@ -106,13 +106,22 @@ export class Scatter extends Component {
 	addLabelsToDataPoints(d, index) {
 		const { labels } = this.model.getDisplayData();
 
-		return d.data.map((datum, i) => ({
-			date: datum.date,
-			label: labels[i],
-			datasetLabel: d.label,
-			class: datum.class,
-			value: isNaN(datum) ? datum.value : datum
-		}));
+		return d.data
+			// Remove datapoints with no value
+			.filter((datum: any) => {
+				if (!datum || datum.value === null) {
+					return false;
+				}
+
+				return true;
+			})
+			.map((datum, i) => ({
+				date: datum.date,
+				label: labels[i],
+				datasetLabel: d.label,
+        class: datum.class,
```

Comment:
This indentation is definitely not intentional 
```suggestion
				class: datum.class,
```


^ that _may_ fix it

Reply:
it's the github conflict resolver